examples/surfaceqg_decaying.jl

```diff
@@ -171,16 +175,16 @@ anim = @animate for j=0:Int(nsteps/nsubs)
 
   cfl = clock.dt * maximum([maximum(vars.u) / grid.dx, maximum(vars.v) / grid.dy])
 
-  if j%(1000/nsubs)==0
+  if j%(500/nsubs)==0
     log1 = @sprintf("step: %04d, t: %.1f, cfl: %.3f, walltime: %.2f min",
           clock.step, clock.t, cfl, (time()-startwalltime)/60)
-    log2 = @sprintf("buoyancy variance diagnostics - B: %.2e, Diss: %.2e",
+    log2 = @sprintf("buoyancy variance: %.2e, buoyancy variance dissipation: %.2e",
               B.data[B.i], Dᵇ.data[Dᵇ.i])
     println(log1)
     println(log2)
   end
 
-  p[1][1][:z] = Array(vars.b)
+  p[1][1][:z] = vars.b
```

Comment:
Should this b field be transposed, since the original field (Line 128) and the last figure (Line 227) use transposed buoyancy fields?

Reply:
This was puzzling me as well! Initially I had here `p[1][1][:z] = vars.b'` but I was getting a plot with the vortex transposed! So I removed the transpose.... Perhaps it's something to do with how Plots.jl handles this array updates... 🤷🏼‍♂️